service/controller/v11/key/key.go

```diff
@@ -53,10 +53,11 @@ const (
 )
 
 const (
-	AnnotationAPIEndpoint = "kvm-operator.giantswarm.io/api-endpoint"
-	AnnotationIp          = "endpoint.kvm.giantswarm.io/ip"
-	AnnotationService     = "endpoint.kvm.giantswarm.io/service"
-	AnnotationPodDrained  = "endpoint.kvm.giantswarm.io/drained"
+	AnnotationAPIEndpoint   = "kvm-operator.giantswarm.io/api-endpoint"
+	AnnotationIp            = "endpoint.kvm.giantswarm.io/ip"
+	AnnotationService       = "endpoint.kvm.giantswarm.io/service"
+	AnnotationPodDrained    = "endpoint.kvm.giantswarm.io/drained"
+	AnnotationVersionBundle = "kvm-operator.giantswarm.io/version-bundle"
```

Comment:
The annotations are a complete mess. This is basically because of legacy and growing pain. I intent to fix this eventually once we have full automated update support on KVM. Then it should be fairly simple to fix this. The annotation added here is as it should be as of our latest standards.